Source/swiftlint/Extensions/Configuration+CommandLine.swift

```diff
@@ -236,7 +236,11 @@ extension Configuration {
         if !visitor.quiet {
             let filesInfo: String
             if visitor.paths.isEmpty || visitor.paths == [""] {
-                filesInfo = "in current working directory"
+                if self != Self.default, includedPaths.isNotEmpty {
```

Comment:
Why is the check `self != Self.default` necessary? Isn't `includedPaths.isNotEmpty` enough?

Reply:
Good point!